### PR TITLE
Develop branch testing

### PR DESCRIPTION
**Pull request by vjrmalla**
changed output text

### CODE REVIEW

File: hello.go [hunk 1]
```diff
@@ -2,5 +2,5 @@ package main
 import "fmt"
 
 func main(){
-	fmt.Println("Hello World, I am new to Go")
+	fmt.Println("develop branch for organisation")
```
Comment (mvjr) [inline hello.go:5]: change text

File: hello.go [hunk 2]
```diff
@@ -2,5 +2,5 @@ package main
 import "fmt"
```
Comment (mvjr) [inline hello.go:3]: remove space

**Review by mvjr — COMMENTED** [review]
check my comments

File: hello.go (hunk 1 above)
Comment (vjrmalla) [reply hello.go:5]: changed

**Pull request merged**